week-4-practise/level-2/exercise-1.js

```diff
@@ -10,13 +10,17 @@ We made a start on this for you here:
 
 var bottle = {
   volume: 0,
-  fill: function() {
+  fill: function () {
+    return this.volume = 100;
     // calling this function should make you bottles volume = 100;
   },
-  drink: function() {
+  drink: function () {
+    return this.volume -= 10;
     // calling this function should decrease your bottles volume by 10;
   },
-  empty: function() {
+  empty: function () {
+    return this.volume === this.volume ? true : false;

```

Comment:
Think there's a typo here - you want to check the volume is zero, not that the volume equals the volume, surely?